content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -5,272 +5,280 @@ weight: 50
 
-* The Istio control plane services (Pilot, Mixer, Citadel) and Kubernetes DNS server must be accessible
-from the VMs. This is typically done using an [Internal Load
-Balancer](https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer).
-You can also use NodePort, run Istio components on VMs, or use custom network configurations,
-separate documents will cover these advanced configurations.
+* Mesh expansion VMs must have access to a DNS server that resolves names to cluster IP addresses. Options
+include exposing the Kubernetes DNS server through an internal load balancer, using a Core DNS
+server, or configuring the IPs in any other DNS server accessible from the VM.
+
+* If you haven't already enabled mesh expansion [at install time with Helm](/docs/setup/kubernetes/helm-install/), you have installed the [Helm client](https://docs.helm.sh/using_helm/). You'll need it to enable mesh expansion for the cluster.
 
 ## Installation steps
 
 Setup consists of preparing the mesh for expansion and installing and configuring each VM.
 
-An example script to help with Kubernetes setup is available as part of the release bundle called
-[`install/tools/setupMeshEx.sh`]({{< github_file >}}/install/tools/setupMeshEx.sh). Check the script content and environment variables supported (like GCP_OPTS).
+### Preparing the Kubernetes cluster for expansion
 
-An example script to help configure a machine is available as part of the release bundle called [`install/tools/setupIstioVM.sh`]({{< github_file >}}/install/tools/setupIstioVM.sh).
-You should customize it based on your provisioning tools and DNS requirements.
+The first step when adding non-Kubernetes services to an Istio mesh is to configure the Istio installation itself and generate the configuration that will allow it to be used by the mesh expansion VMs. To prepare the cluster for mesh expansion, run the following commands on a machine with cluster admin privileges:
 
-### Preparing the Kubernetes cluster for expansion
+1.  Ensure that mesh expansion is enabled for the cluster. If you did not specify `--set global.meshExpansion=true` at
+    install with Helm, you can either `helm upgrade` with the new option (note that this will only work if you originally
+    installed with Helm and Tiller), or you can use `helm template` to update your configuration with the option and reapply
+    with `kubectl`.
+    
+    {{< text bash >}}
+    $ cd install/kubernetes/helm/istio
+    $ helm upgrade --set global.meshExpansion=true istio-system .
+    $ cd -
+    {{< /text >}}
+    
+    {{< text bash >}}
+    $ cd install/kubernetes/helm/istio
+    $ helm template --set global.meshExpansion=true --namespace istio-system . > istio.yaml
+    $ kubectl apply -f istio.yaml
+    $ cd -
+    {{< /text >}}
+    
+    You can either set the option on the command line, as in our examples, or add it to a `.yaml` values file and pass it to
```

Comment:
It's specifically something to consider when using Helm, though, rather than anything else in the doc (and this process requires that you use Helm even if you haven't used it for anything else). Agree it's a bit of an aside though, I'll put it in a note box.